src/test/java/uk/gov/hmcts/reform/pip/data/management/service/schemavalidation/OpaPublicListTest.java

```diff
@@ -0,0 +1,384 @@
+    package uk.gov.hmcts.reform.pip.data.management.service.schemavalidation;
```

Comment:
Should add a unit test for OPA_PUBLIC_LIST to ValidationServiceTest. You can see the example in this PR :) 
https://github.com/hmcts/pip-data-management/pull/476/files#diff-aeb523462c208ae786797d1b1bb7b62e6bec1a37859386a34a686c7bf17fcd40